config/params.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+declare(strict_types=1);
+
+return [
+    'app' => [
+        'brandurl' => '/',
+        'charset' => 'UTF-8',
+        'hero.options' => ['class' => 'hero is-fullheight is-light'],
+        'hero.head.options' => ['class' => 'hero-head has-background-black'],
+        'hero.body.options' => ['class' => 'hero-body is-light'],
+        'hero.container.options' => ['class' => 'container has-text-centered'],
+        'hero.footer.options' => ['class' => 'hero-footer has-background-black'],
+        'hero.footer.column.options' => ['class' => 'columns is-mobile'],
+        'hero.footer.column.left' => 'Left',
+        'hero.footer.column.left.options' => ['class' => 'column has-text-left has-text-light'],
+        'hero.footer.column.center' => 'Center',
+        'hero.footer.column.center.options' => ['class' => 'column has-text-centered has-text-light'],
+        'hero.footer.column.rigth' => 'Rigth',
+        'hero.footer.column.rigth.options' => ['class' => 'column has-text-right has-text-light'],
+        'language' => 'en',
+        'logo' => '/images/yii-logo.jpg',
+        'name' => 'My Project Basic',
+        'navbar.options' => ['class' => 'navbar'],
+        'navbar.brand.options' => ['class' => 'navbar-brand'],
+        'navbar.brand.logo.options' => ['class' => 'navbar-item'],
+        'navbar.brand.title.options' => ['class' => 'navbar-item has-text-light'],
+    ],
+
+    'aliases' => [
+        '@root' => dirname(__DIR__),
+        '@public' => '@root/public',
+        '@images' => '@public/images',
+        '@resources' => '@root/resources',
+        '@basePath' => '@public/assets',
+        '@web' => '/assets',
+        '@npm' => '@root/node_modules',
+        '@layout' => '@root/resources/layout',
+        '@views' => '@root/resources/views',
+        '@src' => '@root/src',
+    ],
+
+    'debugger.enabled' => true,
```

Comment:
In the telegram chat we have discussed about parameters naming convention.
Parameters should be named as `vendor name` slash `package name`, e.g. `yiisoft/yii-debug => [ ... ]`
See https://github.com/yiisoft/yii-debug/pull/46

Reply:
Done.